### PR TITLE
add account page

### PR DESCRIPTION
**Pull request by smartdev0809**

### CODE REVIEW

File: src/pages/account.jsx [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        files: user.avatar || [],
+      });
+  }, [user]);
+  useEffect(() => {
+    setErrorMessage(errorMessage);
+    !loading &&
+      setTimeout(() => {
+        setErrorMessage("");
+      }, 3000);
+  }, [errorMessage]);
```
Comment (dot425) [inline src/pages/account.jsx:58]: plx fix time

File: src/pages/account.jsx [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+      {loading && (
+        <div className="absolute top-0 left-0 w-full h-full">
+          <div className="absolute top-[50%] left-[50%] -translate-x-1/2 -translate-y-1/2 z-[2]">
+            <Loader />
+          </div>
+        </div>
+      )}
+    </>
+  );
+}
```
Comment (pandablue0809) [inline src/pages/account.jsx:300]: plz match coding style

**Review by charmhorse — APPROVED** [review]

**Review by dot425 — APPROVED** [review]

**Review by pandablue0809 — APPROVED** [review]

**Comment by pandablue0809** [comment]
LGTM

**Comment by dot425** [comment]
LGTM

**Comment by charmhorse** [comment]
LGTM

**Pull request merged**